### ISSUE TITLE
Ingress: error endpoints "default-http-backend" not found

### ISSUE THREAD

**Issue by kravciak** [rancher/rancher#33775]
**Rancher Server Setup**
- Rancher version: master-head (2e6926b)
- Installation option: Docker install

**Information about the Cluster**
- Kubernetes version: DO: v1.21.3+rke2r1, local: v1.20.7+k3s1
- Cluster Type (Local/Downstream): both
  - local cluster
  - DO: single node infra provider cluster rke2
  - rke1 works fine

**Describe the bug**
- Ingress is stack at `Initializing` state after creation
-  It creates endpoint, which is not deleted when I delete ingress (kubectl get endpoints)
```
~ kubectl describe ingress theing
Name:             theing
Namespace:        default
Address:          
Default backend:  default-http-backend:80 (<error: endpoints "default-http-backend" not found>)
Rules:
  Host        Path  Backends
  ----        ----  --------
  nip.io      
              /ing   ingress-9c935bddd34775848641d19da57b6859:80 (10.42.25.14:80)
```

**To Reproduce**
- add workload to default project (nginx:alpine)
- create ingress and point to this workload (path: /ing, port: 80, the rest left default)

**Screenshots**
![Screenshot from 2021-07-27 11-31-39](https://user-images.githubusercontent.com/538604/127132132-0aa4115a-4113-4e6a-892b-b856dd2df89b.png)

**Comment by cbron** [comment]
Likely related to: https://github.com/rancher/rancher/issues/30356

**Comment by nickgerace** [comment]
This issue might be a duplicate of the RKE2 one: https://github.com/rancher/rke2/issues/1446

**Comment by deniseschannon** [comment]
@kravciak Can this be reproduced with a RKE1 cluster using the new default ingress backend?

**Comment by nickgerace** [comment]
Spoke with @kravciak @kinarashah and confirmed that this is a duplicate of https://github.com/rancher/rke2/issues/1446, so we will close it after discussing with all parties.

@deniseschannon from Martin's testing that's been relayed to us, as well as Kinara and I checking out the cluster, RKE1 does not encounter the same issue by default.

**Comment by Jono-SUSE-Rancher** [comment]
@nickgerace It is sort of a dupe - however we need to track getting the fix into Rancher as well, so I think we should probably keep this open until the RKE2 issue https://github.com/rancher/rke2/issues/1446 has been confirmed and closed, and then we can confirm and close ours as well.

**Comment by nickgerace** [comment]
I'll put it in Waiting for RC, since this does not require changes from the Rancher team (as far as we are aware).

**Comment by deniseschannon** [comment]
Technically, "Waiting for RC" should be reserved for Waiting for Rancher RC. @Jono-SUSE-Rancher we should clarify where these should go when we're waiting.

**Comment by catastrophe-brandon** [comment]
Note: In attempting to repro this, I'll interpret "add workload to default project (nginx:alpine)" to mean "create a Workload using the EmberUI" because this is what the screenshot depicts.

**Comment by catastrophe-brandon** [comment]
Confirmed (`master-3499046fcdc949c7ab17dbc444dac4c058ab3c52-head`):
1. Ingress does not get stuck Initializing; it eventually becomes active.
2. Endpoint is created
3. Upon delete the ingress goes away.

**Comment by deniseschannon** [comment]
Available as part of RKE2 v1.21.3-rc1+rke2e2
